virginia.clubrunning.org/join.html

```diff
@@ -154,7 +154,7 @@ <h3>Women's team</h3>
                         </tr>
                         <tr>
                             <td>Tuesday</td>
-                            <td>Hard 5mi</td>
+                            <td>Easy 4-6mi</td>
```

Comment:
The sample week on the join page was created in consultation with the workout coordinators, so while I agree that this is probably a good change, I would like to run it by Katie first. I request that you leave this alone for this PR.